draft-ietf-jsonpath-base.md

```diff
@@ -1193,6 +1193,23 @@ Some examples:
 | `{"a":{"b":{5},c:0}}` | `$[?@.b==5 && !@.c]` | `[{"b":{5},c:0}]` | Existence  |
 
 
+#### Examples
+{: unnumbered}
+
+JSON document:
+
+    [3, 5, 1, 2, 4, 6, {"a": "ij"}, {"a": "ik"}]
+
+Queries:
+
+| Query | Result | Result Paths | Comment |
+| :---: | ------ | :----------: | ------- |
+| `$[?@>3.5]` | `5` <br> `4` <br> `6` | `$[1]` <br> `$[4]` <br> `$[5]` | Comparison |
+| `$[?@.a]` | `{"a": "ij"}` <br> `{"a": "ik"}` | `$[6]` <br> `$[7]` | Existence |
+| `$[?@<2 || @.a == "ik"]` | `1` <br> `{"a": "ik"}` | `$[2]` <br> `$[7]` | Logical OR |
+| `$[?@.a =~ "i.*"]` | `{"a": "ij"}` <br> `{"a": "ik"}` | `$[6]` <br> `$[7]` | Regular expression |
```

Comment:
Regex should be `"i[j,k]"` or more generally  `"i[a-z]"` iirc ... thanks for your effort.

Reply:
Why? Suppose my _intention_ was to pick any value beginning with `i`. Is there some value in having one regex rather than another?

Reply:
ahh ... `.*` ... means zero or more arbitrary characters  ... I missed that ... yes you are right.

Reply:
No problem. I thought you had some objection to `.*`.